api/cloud/oc_cloud_manager.c

```diff
@@ -123,7 +123,8 @@ cloud_start_process(oc_cloud_context_t *ctx)
   ctx->retry_count = 0;
   ctx->retry_refresh_token_count = 0;
 
-  if (ctx->store.status == OC_CLOUD_INITIALIZED) {
+  if (ctx->store.status == OC_CLOUD_INITIALIZED &&
```

Comment:
maybe we can check also if the device is in the `RFNORM` state. What do you think ?
```
#include "security/oc_pstat.h"
...
 oc_sec_pstat_t *pstat = oc_sec_get_pstat(ctx->device);
 if (pstat->s != OC_DOS_RFNOP) {
   return;
 }
 ```

Reply:
@jkralik Yes, I aggree. That's a good suggestion. I will try it out.

Reply:
I added the check in the commit https://github.com/iotivity/iotivity-lite/pull/192/commits/124049a76267ef0afe2130dab07bb4c1e49f4c3e